### DISCUSSION TITLE
Could Prisma v3.15.0 work with MySQL 8.0?

### DISCUSSION THREAD

**Discussion by zolbayars** [prisma/prisma#24957] [Q&A]
Hi all, we're upgrading from MySQL 5.7 to 8.0. Yet we're still using v3.15.0 Prisma. Would that be fine or shall we upgrade Prisma beforehand? 🙂

**Comment by ludralph** [comment]
Hi @zolbayars 👋 

Prisma supports MySQL 8.0. See the list of [supported databases](https://www.prisma.io/docs/orm/reference/supported-databases).

We recommend that our users upgrade to the latest version of [Prisma](https://github.com/prisma/prisma/releases/tag/5.17.0) as we constantly ship improvements to the ORM. For example, this [article](https://www.prisma.io/blog/prisma-5-f66prwkjx72s) highlights the performance improvements that came with Prisma 5. We also allowed our users to be able to choose the best [Join](https://www.prisma.io/blog/prisma-orm-now-lets-you-choose-the-best-join-strategy-preview) strategy when working with Prisma.

**Reply by zolbayars** [reply]
Hi @ludralph 👋 

I know Prisma supports MySQL 8.0 🙂. What I wonder is if that specific version of Prisma (v3.15.0) supports it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
